app/helpers/question_attribute_helper.rb

```diff
@@ -20,16 +19,6 @@ def numeric_question_html_attributes(question)
       "data-calculated": question.fields_to_add.to_json,
     }
   end
-
-  def conditional_html_attributes(question)
```

Comment:
Can't actually remove this until we also change the Other Household Members conditional questions but after that we can.